proto/src/determined/task/v1/task.proto

```diff
@@ -47,3 +49,15 @@ message Task {
   // List of Allocations.
   repeated Allocation allocations = 4;
 }
+
+// Task allocations with an associated Experiment.
+message ExperimentAllocation {
```

Comment:
this should just be a go struct, not be in protos

Reply:
✅